### DISCUSSION TITLE
Neverball Extreme

### DISCUSSION THREAD

**Discussion by Adacrop-The-Great** [Neverball/neverball#313] [Show and tell]
After a few months, Neverball Extreme has been updated! [Click here to view the patch notes of the May 6, 2026 update and download.](https://github.com/Adacrop-The-Great/Neverball-Extreme/releases/tag/v3.1.0)

**Comment by Lillyfish99** [comment]
I have never seen this project. Why was I mentioned?

**Reply by parasti** [reply]
There was bot spam here. I assume you received a notification via email as everyone else. You can read it there.

**Reply by Lillyfish99** [reply]
Ok, thank you for the information!

**Comment by Adacrop-The-Great** [comment]
Another update has been added. Click the link in the initial post for the patch notes and to download the updated franchise.

**Reply by Adacrop-The-Great** [reply]
Yet another update has been added
